sql/moz-fx-data-shared-prod/accounts_backend_external/nonprod_accounts_v1/query.sql

```diff
@@ -0,0 +1,22 @@
+SELECT
+  *
+FROM
+  EXTERNAL_QUERY(
+    "moz-fx-fxa-nonprod.us.fxa-rds-nonprod-stage-fxa",
+    """SELECT
```

Comment:
This query cannot be made incremental because this table does not have a "last modified" column.
I'm going to evaluate Datastream for incremental imports in parallel to this work in case full dumps turn out to be problematic.

Reply:
Snapshot is fine for this table, we can track history in a derived table.

Reply:
It's my understanding that FxA hard deletes an account's database records when the user clicks "Delete account" in their profile settings, which will be something to keep in mind for downstream uses of this data or if we adopt any sort of incremental approach.

Currently we're taking Firefox Account deletion events from the logs as the cue to shred related FxA data in BigQuery.  Since FxA hard deletes records and we're overwriting this table every time it's effectively self-shredding, but if we change it to be incremental then we'd need to configure it for shredding.